var/spack/repos/builtin/packages/arrayfire/package.py

```diff
@@ -64,10 +61,34 @@ def cmake_args(self):
             self.define_from_variant('AF_BUILD_CUDA', 'cuda'),
             self.define_from_variant('AF_BUILD_FORGE', 'forge'),
             self.define_from_variant('AF_BUILD_OPENCL', 'opencl'),
+            self.define_from_variant('BUILD_TESTING', 'tests'),
```

Comment:
Since testing (shouldn't) affect the installed package, it's better not to have a variant for it.
```suggestion
            self.define('BUILD_TESTING', run_tests),
```